File: c_Programming/Assingements/Assignment 14/pass by address/Employee.c
#include<stdio.h>
typedef struct Employee
{
	int id;
	char name[20];
	double salary;
}Employee;
void storeEmployeedata(Employee*);
void display(Employee*);
void main()
{
	 Employee emp1,emp2;
	
	storeEmployeedata(&emp1);
	display(&emp1);
	
	storeEmployeedata(&emp2);
	display(&emp2);
	
}
void storeEmployeedata(Employee* emp)
{
	printf("\n\nEnter the Employee Data Id , name and salary :");
	scanf("%d%s%lf",&emp->id,emp->name,&emp->salary);
	
}
void display(Employee*emp)
{
	printf("\nEmployee Id :%d \nEmployee name :%s \nEmployee salary :%.2lf",emp->id,emp->name,emp->salary);
}

File: Java_Module/demo java/Scanner/Employee.java
import java.util.Scanner;

class Employee{
    int id;
    String name;
    double salary;
	//set 
	void setId(int i){
	this.id=i;
	}
	void setName(String n){
	this.name=n;
	}
	void setSalary(double s){
	this.salary=s;
	}
	//get
	int getId(){
	return this.id;
	}
	String getName(){
	return this.name;
	}
	double getSalary(){
	return this.salary;
	}
	//display
	void display()
       {
	System.out.println("Employee Id : "+this.id);
        System.out.println("Employee Name : "+this.name);
        System.out.println("Employee Salary : "+this.salary);
	}


}

class EmployeeData{
    public static void main(String args[])
    {
        System.out.println("Employee Data");

        Scanner s=new Scanner(System.in);
	
	Employee emp=new Employee();
	System.out.println("Enter a HR Id: ");
	int i=s.nextInt();
	emp.setId(i);
	
	System.out.println("Enter a HR Name : ");
	s.nextLine();
	String n=s.nextLine();
	emp.setName(n);
	
	System.out.println("Enter a Salary :");
	double sa=s.nextDouble();
	emp.setSalary(sa);
	
	emp.display();


            }
}